components/Buttons/src/ColorThemer/MDCContainedButtonColorThemer.m

```diff
@@ -27,6 +27,9 @@ + (void)applySemanticColorScheme:(nonnull id<MDCColorScheming>)colorScheme
   [button setTitleColor:colorScheme.onPrimaryColor forState:UIControlStateNormal];
   [button setTitleColor:[colorScheme.onSurfaceColor colorWithAlphaComponent:0.38f]
                forState:UIControlStateDisabled];
+  [button setImageTintColor:colorScheme.onPrimaryColor forState:UIControlStateNormal];
+  [button setImageTintColor:[colorScheme.onSurfaceColor colorWithAlphaComponent:0.38f]
+                   forState:UIControlStateDisabled];
```

Comment:
imageTintColor == titleColor, LGTM.
It is slightly suspicious that .normal uses onPrimary and .disabled uses onSurface, but I will assume that this is by design.